src/main/java/org/carlmontrobotics/commands/TeleopEffector.java

```diff
@@ -75,6 +78,6 @@ public void end(boolean interrupted) {
   // Returns true when the command should end.
   @Override
   public boolean isFinished() {
-    return false;
+    return !intake.intakeDetectsNote() && !intake.outtakeDetectsNote();
```

Comment:
Default commands' `isFinished()` method should always return false, because they should only end when interrupted by another command that needs to use the subsystem.

Reply:
still gotta fix this

Reply:
I think i did change it to false